include/Core/Controller.php

```diff
@@ -161,6 +162,31 @@ public function load_taxonomy_term() {
 		die;
 	}
 
+	/**
+	 * Ajax call back function to load term meta by term.
+	 *
+	 * @since 6.1.0
+	 */
+	public function load_term_metas() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [load_term_metas accesses the super-global variable $_GET.](https://app.codacy.com/app/BulkWP/bulk-delete/pullRequest?prid=1939930)